### DISCUSSION TITLE
Reintroduce internal browser

### DISCUSSION THREAD

**Discussion by SanderJSA** [spacecowboy/Feeder#136] [General]
Hi !

On older versions of Feeder we used to be able to use the internal browser instead of being directly sent to the main browser.
This was very useful as I could disable Javascript, which would allow me to read article without clutter most of the time.
Additionally opening on the main browser floods my open tabs, after a week I'm at 50 tabs despite my best efforts to go back once I'm done.
This feels like a massive downgrade for my usage, I was wondering if it was possible to add it back ?

Maybe it's still supported but I couldn't get it to work, despite asking it to open items with Reader and open links with custom tab in the Settings.

To replicate my issue:
feed: https://news.ycombinator.com/rss
Open any entry, click on Comments

Previous version would open comments in Feeder
Current version opens it in the browser

Once again thanks for all the work you've done so far !
Cheers

**Comment by spacecowboy** [comment]
Custom tab IS your browser. But it is separate from all your normal tabs.

Install a browser which support custom tabs maybe. Firefox works. Brave works.

**Comment by SanderJSA** [comment]
I'm using Brave, updated it to the latest version, the issue still remains unfortunately.
Brave 1.60.118, Chromium 119.0.6045.163.

**Reply by spacecowboy** [reply]
See https://github.com/brave/brave-browser/issues/30241

This might be a setting you might need to set in Brave.

**Reply by SanderJSA** [reply]
Nice find !
Enabling "Open external links in Brave" opens links in a webview instead of a new tab which fixes my issue.
Thank you so much for your help !
